app/controllers/shows_controller.rb

```diff
@@ -58,4 +58,17 @@ def destroy
     Show.find(params[:id]).destroy
     redirect_to shows_path
   end
+
+  # migrating from protected attr to strong param
+  # standard found here: https://www.fastruby.io/blog/rails/upgrades/strong-parameters-migration-guide.html
+  private
+  def permit_new_show
+    params.permit :season
+    params.require :season
```

Comment:
try taking this out

Reply:
resolved, threw an error on the null case. replaced with fetch see recent updates